lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -7,7 +7,7 @@
 module KubernetesDeploy
   class KubernetesResource
     attr_reader :name, :namespace, :context
-    attr_writer :type, :deploy_started_at
```

Comment:
I've removed `type` and replaced it with `kind` so that we match the Kubernetes nomenclature.